### PR TITLE
[Test] Fix test on JoinColumns multi line

### PR DESCRIPTION
**Pull request by samsonasik**

### CODE REVIEW

**Comment by samsonasik** [comment]
@TomasVotruba it seems there is phpstan error:

```bash
  Service of type Symplify\PHPStanRules\Rules\Missing\CheckReferencedClassInAnnotationRule: Service of type Symplify\SimplePhpDocParser\SimplePhpDocParser needed by $simplePhpDocParser in __construct() not found. Did you add it  
   to configuration file? 
```

**Comment by samsonasik** [comment]
PHPStan fixed by add: 

```neon
- vendor/symplify/phpstan-rules/config/services/services.neon
```

to phpstan.neon

File: tests/Set/DoctrineORM29Set/Fixture/join_columns.php.inc [hunk 1]
```diff
@@ -25,7 +25,7 @@ use Doctrine\ORM\Mapping as ORM;
 
 class AnEntity
 {
-    #[\Doctrine\ORM\Mapping\JoinColumns(['(name="entity_id", referencedColumnName="id")', '(name="entity_type", referencedColumnName="entity_type")', '}'])]
+    #[\Doctrine\ORM\Mapping\JoinColumns(['(name="entity_id", referencedColumnName="id")', '(name="entity_type", referencedColumnName="entity_type")'])]
```
Comment (TomasVotruba) [inline tests/Set/DoctrineORM29Set/Fixture/join_columns.php.inc:28]: :clap:

**Comment by TomasVotruba** [comment]
It should be added by using this package https://packagist.org/packages/phpstan/extension-installer instead

The `service.neon` is then added by phpstan itself

**Comment by samsonasik** [comment]
I think what you meant is this https://github.com/rectorphp/extension-installer as that is what we have.

**Comment by samsonasik** [comment]
or this https://github.com/symplify/phpstan-extensions/blob/main/config/config.neon ?

**Comment by samsonasik** [comment]
I tried under `extra` config in composer.json: 

```json
    "extra": {
        # ...
        "phpstan": {
            "includes": [
                "vendor/symplify/phpstan-rules/config/services/services.neon"
            ]
        },
        # ...
    },
```

it seems not works.

**Comment by TomasVotruba** [comment]
Nono, I meant https://packagist.org/packages/phpstan/extension-installer 
It's for PHPStan.

It's missing here: https://github.com/rectorphp/rector-doctrine/blob/main/composer.json#L9

**Comment by samsonasik** [comment]
ok, implemented at https://github.com/rectorphp/rector-doctrine/pull/10

**Comment by TomasVotruba** [comment]
Thank you

**Pull request merged**